spec/takeaway_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:order) { double :order, as_hash: {'Spring Rolls' => 2, 
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.